mm-bot/src/models/order.py

```diff
@@ -37,3 +45,21 @@ def get_int_amount(self) -> int:
 
     def get_int_fee(self) -> int:
         return int(self.fee)
+
+    def summary(self) -> str:  # TODO set a better name
```

Comment:
I think this is an OK name 
```suggestion
    def summary(self) -> str:
```